cli/config/configfile/file.go

```diff
@@ -36,6 +36,10 @@ type ConfigFile struct {
 	Filename             string                      `json:"-"` // Note: for internal use only
 	ServiceInspectFormat string                      `json:"serviceInspectFormat,omitempty"`
 	ServicesFormat       string                      `json:"servicesFormat,omitempty"`
+	HTTPProxy            string                      `json:"httpProxy,omitempty"`
```

Comment:
This could be misleading. Someone could look at these and think they're for all client connections and not just for use in the build as build-args. Could we prefix these with BuildArg, or something, to make this clear?